README.md

```diff
@@ -419,3 +419,22 @@ In addition to the ecosystem docs, each package has it's own README.md and `docs
 - fxa-profile-server [README](./packages/fxa-profile-server/README.md) / [docs/](./packages/fxa-profile-server/docs)
 - fxa-shared [README](./packages/fxa-shared/README.md)
 - fxa-support-panel [README](./packages/fxa-support-panel/README.md)
+
+---
+
+### Documentation for Scripts
+
+#### \_scripts/legal-md-to-pdf.sh
+
+##### Purpose
+
+This bash script allows for the conversion of markdown files to pdfs. The purpose is to create a way for legal documents to be easily converted and made available for end-users.
+
+##### Usage
+
+The script requires that you have pandoc and LaTex installed on your machine.
+The script will check that pandoc is installed before asking you for the file or directory on your machine that you would like to be converted. If a directory is provided, then all markdown files within that folder will be convered. If a single file is provided, only that file will be converted.
+The resulting output will be stored within `assets/legal`.
+Example:
+file provided to convert: `/Users/test/github/mozilla/legal-docs/firefox_cloud_services_ToS/en-US.md`
```

Comment:
The cloud services PDFs in this PR should help as general output samples for review.

But, IIRC, FPN has a separate product-specific set of [terms](https://www.mozilla.org/about/legal/terms/firefox-private-network) & [privacy notice](https://www.mozilla.org/privacy/firefox-private-network) that we'll want to link to for download in emails. We may want to generate those also / instead, because I assume they'll need to be reviewed by legal too.

Reply:
Yes that is correct, and is planned. The samples provided were for legal to determine if they were satisfied with the conversion process. We are waiting on the product specific documents to be uploaded.